README.md

```diff
@@ -43,7 +37,7 @@ implementation(enforcedPlatform("io.netty:netty-bom")) {
 // Overrides the default Jackson databind module version with a specific version
 implementation ("com.fasterxml.jackson.core:jackson-databind") {
     version {
-        strictly '2.13.2.1'
+        strictly '2.13'
```

Comment:
There is no 2.13 version, consider using the latest as of today: 2.13.3: https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind